app/middleware/fedache.rb

```diff
@@ -0,0 +1,37 @@
+require 'fixed_length/parser'
+require 'fixed_length/column'
+require 'faraday_middleware/response_middleware'
+
+class Fedache < FaradayMiddleware::ResponseMiddleware
+  extend FixedLength::Column
+
+  column :routing_number, length: 9
```

Comment:
I think these column definitions should be in a separate module or an instance of something like a `FixedLength::Format` class. I don't think middleware should define file format like this.

mea culpa.